mlflow/evaluation.py

```diff
@@ -0,0 +1,354 @@
+from typing import Dict, Union
+import entrypoints
+import warnings
+import mlflow
+import hashlib
+import time
+import numpy as np
+import pandas as pd
+import pickle
+from mlflow.exceptions import MlflowException
+from mlflow.utils.file_utils import TempDir
+from mlflow.entities import Metric
+from mlflow.tracking.artifact_utils import get_artifact_uri, _download_artifact_from_uri
+
+
+class EvaluationMetrics(dict):
+    pass
+
+
+class EvaluationArtifacts:
+
+    def __init__(self, location, content=None):
+        self._content = content
+        self._location = location
+
+    def load_content_from_file(self, local_artifact_file):
```

Comment:
Awesome! For the initial implementation, I'm assuming we'll create subclasses of `EvaluationArtifact` for images, CSVs, etc?

Reply:
Yes.